exla/lib/exla.ex

```diff
@@ -184,6 +176,45 @@ defmodule EXLA do
 
   @behaviour Nx.Defn.Compiler
 
+  @doc """
+  Sets the global defn options to the EXLA compiler with the
+  preferred client based on the available platforms.
+
+  This function is typically invoked at the top of scripts
+  and code notebooks which might be potentially executed from
+  multiple platforms. Do not invoke this function during runtime,
+  as it changes the `Nx.Defn` options globally. If you have a
+  specific client that you want to use throughout your project,
+  use configuration files instead:
+
+      config :nx, :default_defn_options, [compiler: EXLA, client: :cuda]
```

Comment:
I'm thinking here if maybe : global_defn_options or just :defn_options could work here, since Application.get_env is global by definition

Reply:
+1

Reply:
Actually it's nice that it corresponds to to `Nx.Defn.global_default_options`, more precisely it's `global_default_defn_options` and we strip the global part exactly as you said :)